### DISCUSSION TITLE
Release v2022.3 - Wed 21/09/2022

### DISCUSSION THREAD

**Discussion by arantzaetxebarria** [INSPIRE-MIF/helpdesk-validator#850] [Announcements]
Dear Community,

The next release of the INSPIRE Reference Validator has been scheduled to take place on Wednesday 21/09/2022 from 14:30 to 16:30.
We will inform you of the release status and the Production instance availability.
In the meantime, if you experience issues in the Production instance, we kindly invite you to use the staging instance instead.

We will keep you posted on the service status.

Kind regards,

**Comment by jenriquesoriano** [comment]
Dear Community, 

just inform that the v2022.3 release has been deployed successfully in the [production instance](https://inspire.ec.europa.eu/validator/home/index.html).
For further details, please visit the [release notes](https://github.com/INSPIRE-MIF/helpdesk-validator/releases/tag/v2022.3).

Best regards,
